scripts/borg.exe.spec

```diff
@@ -33,33 +33,53 @@ a = Analysis([os.path.join(basepath, 'src', 'borg', '__main__.py'), ],
              win_private_assemblies=False,
              cipher=block_cipher)
 
+pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
+
+# For macOS we build a directory-based binary to get around repeated
+# lib validations. See borgbackup/borg#5185 and
 if sys.platform == 'darwin':
     # do not bundle the osxfuse libraries, so we do not get a version
     # mismatch to the installed kernel driver of osxfuse.
     a.binaries = [b for b in a.binaries if 'libosxfuse' not in b[0]]
 
-pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
-
-exe = EXE(pyz,
-          a.scripts,
-          a.binaries,
-          a.zipfiles,
-          a.datas,
-          name='borg.exe',
-          debug=False,
-          strip=False,
-          upx=True,
-          console=True)
+    exe = EXE(pyz,
+            a.scripts,
+            exclude_binaries=True,
+            name='borg.exe',
+            debug=False,
+            strip=False,
+            upx=False,
+            console=True)
 
-if False:
-    # Enable this block to build a directory-based binary instead of
-    # a packed single file. This allows one to easily look at all included
-    # files (e.g. without having to strace or halt the built binary
-    # and introspect /tmp).
     coll = COLLECT(exe,
-                   a.binaries,
-                   a.zipfiles,
-                   a.datas,
-                   strip=False,
-                   upx=True,
-                   name='borg-dir')
+                a.binaries,
+                a.zipfiles,
+                a.datas,
+                strip=False,
+                upx=False,
+                name='borg-macosx64')
```

Comment:
guess the name could be computed in advance for all platforms using sys.platform.
also we could do the single-file and the directory approach for all platforms.

the pyinstaller run is relatively quick, considering the platform testing runs for hours on some platforms.

Reply:
or, maybe even simpler, use `borg-dir` here also (see non-darwin64 code).

Reply:
thanks for fixing.